firedrake/function.py

```diff
@@ -106,16 +106,22 @@ def ufl_id(self):
         return self.uid
 
     @utils.cached_property
-    def _split(self):
+    def _subfunctions(self):
         return tuple(CoordinatelessFunction(fs, dat, name="%s[%d]" % (self.name(), i))
                      for i, (fs, dat) in
                      enumerate(zip(self.function_space(), self.dat)))
 
     @PETSc.Log.EventDecorator()
-    def split(self):
+    def subfunctions(self):
```

Comment:
So I think that a `subfunctions` property is better, and this could be made clear in the `FutureWarning`, but I will raise it in the Firedrake meeting this Wednesday to discuss with the other developers.